modules/cli-krew-install-plugin.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/installing-cli-plugins-krew.adoc
+
+:_content-type: PROCEDURE
+[id="cli-krew-install-plugin_{context}"]
+= Installing a CLI plug-in with Krew
+
+You can install a plug-in to the OpenShift CLI (`oc`) with Krew.
```

Comment:
Global comment: I was trying to find a better resource on prepositions to use with "install". I could not find specific guidance, yet a couple of examples from IBM Style under [Introducing the procedure](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#introducing-the-procedure), use _install something "on"_
```suggestion
You can install a plug-in on the OpenShift CLI (`oc`) with Krew.
```

Reply:
@abhatt-rh thanks for the suggestion! Thinking about it more, I think that "for" is the most accurate word to use, so I will update to use this.

Based on some wording used in the other section on CLI plug-ins: https://53368--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/extending-cli-plugins.html#cli-installing-plugins_cli-extend-plugins